content/docs/tasks/traffic-management/secure-ingress-sds/index.md

```diff
@@ -0,0 +1,488 @@
+---
+title: Securing Gateways with SDS
+description: Describes how to configure Istio to expose a service outside of the service mesh, over TLS or Mutual TLS, using SDS.
+weight: 31
+keywords: [traffic-management,ingress,sds]
+---
+
+The [Securing Gateways with HTTPS](/docs/tasks/traffic-management/secure-ingress) task describes how to configure an ingress
+gateway to expose an HTTP endpoint of a service to external traffic with either simple or mutual TLS.
+This task shows you how to do the same, but using an ingress gateway agent to provision key/cert and root cert to ingress gateway dynamically.
```

Comment:
Please expand all uses of "cert" to certificate.

Reply:
Done.